### DISCUSSION TITLE
404 instead of defaulting to index.html

### DISCUSSION THREAD

**Discussion by mciolkosz** [community/community#22686] [Other Feature Feedback, Questions, & Ideas]
<p>Hi all,</p>
<p>I’m having an issue where when I go to my site at <a href="https://bspfsystems.org/YamlConfiguration/" rel="nofollow noopener">https://bspfsystems.org/YamlConfiguration/</a>, it displays a 404 instead of loading index.html. I’ve checked a number of other posts, some of which didn’t have the same setup as me. I’ve also tried steps like clearing my cache and even going so far as to try in a browser on a device that had never been to the site before.</p>
<p>There was one in particular (<a href="https://github.community/t/my-github-page-doesnt-redirect-to-index-html/10367/19">this one</a>) that mentioned the README.md file. While my repository does have a README, it is not in the gh-pages branch, which is where my Page is published from.</p>
<p>Any suggestions or help would be great. Thanks.</p>

**Comment by thomasjo** [comment, marked as answer]
<p>I just checked your site, and it seems to work as expected now. I’m not getting any 404 errors at least.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/5/50e49a35324a30b4293dd0de291e13a5256f8811.png" data-download-href="/uploads/short-url/bxBYvBvFiFJuw5NwCbnXVjP9YEF.png?dl=1" title="image" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/3622/181072591-0da04e20-86de-4400-bf51-296059656fe1.png" alt="image" data-base62-sha1="bxBYvBvFiFJuw5NwCbnXVjP9YEF" width="625" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/5/50e49a35324a30b4293dd0de291e13a5256f8811_2_625x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/5/50e49a35324a30b4293dd0de291e13a5256f8811_2_937x750.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/5/50e49a35324a30b4293dd0de291e13a5256f8811.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/5/50e49a35324a30b4293dd0de291e13a5256f8811_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1169×934 88.5 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by mciolkosz** [comment]
<p>I blame the systems, and they know who to fear, and who they can mess with…</p>
<p>(Yes, working for me now…)</p>
